### DISCUSSION TITLE
versification requirements

### DISCUSSION THREAD

**Discussion by jtauber** [bible-technology/scripture-burrito#243] [Next]
discussion for #212 

It sounds like we need a standard set of versifications (even if a tiny set) that can be referred to by name either to say "we're using this named versification scheme" OR "we're using a custom versification scheme and here is its delta from this named versification scheme" OR "we're using a custom versification scheme and here is its delta from the custom versification scheme used in another resource".

Is the third case restricted to resources from which this burrito is derived? (in the formal sense of a derived burrito)

Is the Copenhagen Alliance the only mechanism for defining the standard set of versifications? 

So it seems like our requirements are:

1. a way of referring to standard versifications by name / some identifier
2. a way of expressing deltas between versifications schemes
3. a way of referring to other resources that use a versification scheme from which we want to give deltas

Is there an issue that 3 might require the resource to be retrievable? Would it be better to always inline any non-standard versification schemes to avoid this?

**Comment by jonathanrobie** [comment]
The versification group decided that we will include basic metadata in the JSON format - what generated the versification file and when, etc. 

We will not compute diffs from standard named versification schemes, we will map to and from org, which is available on the Internet.  If you are scrolling N translations, use org for each as the basis for scrolling.

This notebook shows how to use versification:

https://github.com/Copenhagen-Alliance/versification-specification/blob/master/versification-mappings/VersificationMappings.ipynb

**Comment by jtauber** [comment]
Can we agree the following is true:

> All deltas will be expressed against **org** and no other versification system. They may have _originally_ been expressed as deltas to some other versification system but, for the purposes of generating a burrito, the delta against **org** is calculated and that is all that can be in a burrito.

**Comment by jonathanrobie** [comment]
Yes, I think that is the best approach.
